examples/airports.html

```diff
@@ -17,30 +17,25 @@
 
 <body>
     <div id="map"></div>
-    <link rel="stylesheet" href="../L.Control.Layers.Tree.css" crossorigin=""/>
+    <link rel="stylesheet" href="../L.Control.Layers.Tree.css"/>
     <script src="../L.Control.Layers.Tree.js"></script>
 
     <script type="text/javascript">
         var center = [40, 0];
 
         var osm = L.tileLayer(
-            '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-            {attribution: '© OpenStreetMap contributors'}
-        );
-
-        var osmBw = L.tileLayer(
```

Comment:
Better than removing this entry, replace it with other from OSM, like `https://tile.openstreetmap.de/{z}/{x}/{y}.png`